rsconnect/environment.py

```diff
@@ -45,6 +45,9 @@ def detect_environment(dirname, force_generate=False, compatibility_mode=False,
                       or pip_freeze())
 
     if result is not None:
+        if conda_mode and result['package_manager'] != 'conda':
+            return {'error': 'Conda was requested but no Conda environment was found.'}
```

Comment:
`Conda was requested but no activated Conda environment was found. See "conda activate --help" for more information.`

Reply:
Updated.